reorganize-2019-01.md

```diff
@@ -0,0 +1,86 @@
+# Reorganization for rstudio::conf 2019-01
+
+The outline below is for day 1 only.
+
+## 01) Two Ways to Teach
+
+-   Introductions
+-   Comparison with movie/play (add comedy improv)
+-   Tell participants there's lots of research (much of which people don't know/act on)
+-   Schedule (for both days)
+-   Survey distinguishing teacher-centered from learner-centered
+    -   Explain that active learning creates a real-time feedback loop in class
+-   Introduce cognitive architecture diagram (revisit it several times)
+    -   Teaching is "load into short-term memory and keep it there long enough to be transferred"
```

Comment:
I think teaching is more like "load into short-term memory and then help them transfer it to long term" (with exercises and feedback).  (I also like the implied statement that since transfer happens in the students head, the teacher can't do it for them.